website/docs/r/api_management.html.markdown

```diff
@@ -243,9 +281,9 @@ A `security` block supports the following:
 
 -> **Note:** This property has been deprecated in favour of the `enable_frontend_tls11` property and will be removed in version 2.0 of the provider.
 
-* `disable_triple_des_ciphers` - (Optional) Should the `TLS_RSA_WITH_3DES_EDE_CBC_SHA` cipher be disabled for alL TLS versions (1.0, 1.1 and 1.2)? This property was mistakenly inverted and `true` actually enables it. Defaults to `false`.
+* `disable_triple_des_ciphers_enabled` - (Optional) Should the `TLS_RSA_WITH_3DES_EDE_CBC_SHA` cipher be disabled for alL TLS versions (1.0, 1.1 and 1.2)? This property was mistakenly inverted and `true` actually enables it. Defaults to `false`.
 
--> **Note:** This property has been deprecated in favour of the `enable_triple_des_ciphers` property and will be removed in version 2.0 of the provider.
+-> **Note:** This property has been deprecated in favour of the `triple_des_ciphers_enabled` property and will be removed in version 2.0 of the provider.
```

Comment:
It looks like this property already got removed? In which case we can drop it from the documentation.

Reply:
True, I removed it.